docs/en/cloud/AWS-EMR.md

```diff
@@ -168,10 +159,7 @@ Using this boostrap script, Alluxio is installed in `/opt/alluxio/` by default.
 Hive and Presto are already configured to connect to Alluxio.
 The cluster also uses AWS Glue as the default metastore for both Presto and Hive.
 This will allow you to maintain table definitions between multiple runs of the Alluxio cluster.
-
 The default Alluxio worker is allocated a third of the instance's available memory.
```

Comment:
I feel like this should say

"**By** default the the Alluxio worker is allocated **one** third of the instance's maximum available memory."